person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(id, age, name: 'Unknown', parent_permission: true)
+    @id = id
```

Comment:
**OPTIONAL** - I suggest you make the Id a randomly generated number between 1 and 1000. This would make assigning ID to a person easier. Please fix this 🙏